src/main/java/ch/njol/skript/bukkitutil/ComponentWrapper.java

```diff
@@ -0,0 +1,106 @@
+package ch.njol.skript.bukkitutil;
+
+import ch.njol.skript.util.ItemSource;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.converter.Converter;
+
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+
+/**
+ * Used for storing the component and/or the item.
```

Comment:
this doc is not sufficient. What is this class meant to do? what role does it play? Why would I want to use it?